pose_estimator/src/pose_estimator.cpp

```diff
@@ -204,6 +195,11 @@ bool PoseEstimator::spin()
 
 void PoseEstimator::insCallback(const auv_interfaces::StateStamped::ConstPtr& ins_msg)
 {
+  if (!hardware_has_sent_first_message_)
```

Comment:
@patapufate what if this never happens?

Reply:
Changed. Instead of waiting for first message from hardware, the update timer period is configured a larger time. 
1d4ad23